### PR TITLE
fix: Go to select token when clicking disabled input

### PR DESCRIPTION
**Pull request by jorbuedo**
https://emurgo.atlassian.net/browse/YOMO-939

### CODE REVIEW

**Review by banklesss — APPROVED** [review]

File: apps/wallet-mobile/src/features/Swap/common/AmountCard/AmountCard.tsx [hunk 1]
```diff
@@ -60,7 +60,7 @@ export const AmountCard = ({
         {label != null && <Text style={[styles.label, !isEmptyString(error) && styles.labelError]}>{label}</Text>}
```
Comment (stackchain) [inline apps/wallet-mobile/src/features/Swap/common/AmountCard/AmountCard.tsx:61]: maybe, (p.s dunno if the exactly same thing)
```
const handleOnPress = () => noTokenSelected ? navigateTo() : focusInput()
```

Comment (jorbuedo) [reply apps/wallet-mobile/src/features/Swap/common/AmountCard/AmountCard.tsx:61]: Would do the same thing, just moves the condition to the render or the on press

Comment (stackchain) [reply apps/wallet-mobile/src/features/Swap/common/AmountCard/AmountCard.tsx:61]: if it is the same thing I'd say to implement as handle.. become a little more readable.

File: apps/wallet-mobile/src/features/Swap/common/SwapFormProvider.tsx [hunk 2]
```diff
@@ -325,12 +325,14 @@ const swapFormReducer = (state: SwapFormState, action: SwapFormAction) => {
     switch (action.type) {
       case SwapFormActionType.SellTouched:
         draft.sellQuantity.isTouched = true
+        draft.sellQuantity.displayValue = ''
```
Comment (stackchain) [inline apps/wallet-mobile/src/features/Swap/common/SwapFormProvider.tsx:328]: Did product request it? That's sad. We were recovering from any inputted number.

Comment (jorbuedo) [reply apps/wallet-mobile/src/features/Swap/common/SwapFormProvider.tsx:328]: This happens before the calculations, resets the field, but then it's calculated by the swap state and we get the value. It just removes an edge case that was causing some confusion.

**Review by stackchain — APPROVED** [review]

**Pull request merged**